tls/tests/test_config.py

```diff
@@ -40,36 +40,6 @@ def test_tags_remote():
     assert c._tags == ['name:foo', 'server_hostname:www.google.com', 'server:www.google.com', 'port:443']
 
 
-def test_cert():
```

Comment:
These tests are no longer relevant, since the responsibility of ensuring the certs are correct should be abstracted to the wrapper.